GUIDE.md

```diff
@@ -190,7 +190,7 @@ the following:
 All of these things can be toggled using various flags provided by ripgrep:
 
 1. You can disable all ignore-related filtering with the `--no-ignore` flag.
-2. Hidden files and directories can be searched with the `--hidden` flag.
+2. Hidden files and directories can be searched with the `--hidden` (`-.` for short) flag.
```

Comment:
Please make sure changes follow the style of the surrounding text. :-)

Namely, please wrap text so that each line fits in 79 columns (inclusive).

Reply:
Done, I didn't notice :smile: 

And thanks for this great tool.